Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
```

Comment:
@MikeMcQuaid: As far as I understand, if this argument is not specified, `launchd` won't be listening to this socket for activity. The `--inactivity-timeout` flag in conjunction with this argument is what makes `rtags` run on-demand.